From 1f3a869a51cb142f1aa9281b2af497ed888a4e34 Mon Sep 17 00:00:00 2001
From: tstegema <tstegema@2eb42a87-3827-0410-b767-dc15055982e4>
Date: Wed, 25 Jul 2018 18:39:50 +0000
Subject: [PATCH 314/462] Mantis 0051292: ConveyorDev: integrate bhs server
 into platform build

added definitions for JarOpcUaStack, JarNettyAll, JarCommonsIo, JarCommonsConfiguration, JarXStream
build BhsServer on "xmk java"

git-svn-id: svn://localhost/SelfServiceBuild/trunk@544 2eb42a87-3827-0410-b767-dc15055982e4
---
 SelfServiceBuild/Build/java/AntProperties.xml | 17 ++++++++++++++++-
 SelfServiceBuild/Build/java/build.xml         |  3 ++-
 2 files changed, 18 insertions(+), 2 deletions(-)

diff --git a/SelfServiceBuild/Build/java/AntProperties.xml b/SelfServiceBuild/Build/java/AntProperties.xml
index 04d0694..4f0e623 100644
--- a/SelfServiceBuild/Build/java/AntProperties.xml
+++ b/SelfServiceBuild/Build/java/AntProperties.xml
@@ -170,7 +170,6 @@
 <property name="JarCLI"                     value="${DIR_GLOBALS}/ThirdParty/commandlineInterface/1.2/commons-cli-1.2.jar"/>
 
 <!-- Sonstiges -->
-<property name="JarCommonsConfiguration"    value="${DIR_GLOBALS}/ThirdParty/jakarta/commons/configuration/1.0/commons-configuration-1.0.jar"/>
 <property name="JarDom4J"                   value="${DIR_GLOBALS}/ThirdParty/dom4j/1.3/dom4j.jar"/>
 <property name="JarFinJ"                    value="${DIR_GLOBALS}/ThirdParty/finj/finj.jar"/>
 <property name="JarIText"                   value="${DIR_GLOBALS}/ThirdParty/iText/1.0/iText.jar"/>
@@ -317,6 +316,22 @@
 <!-- aopalliance -->
 <property name="JarAopalliance"             value="${DIR_GLOBALS}/ThirdParty/aopalliance/1.0/aopalliance-1.0.jar"/>
 
+
+<!-- OPC -->
+<property name="JarOpcUaStack"              value="${DIR_GLOBALS}/ThirdParty/OpcUaStack/1.02.337.8/OpcUaStack-1.02.337.8.jar"/>
+
+<!-- Netty -->
+<property name="JarNettyAll"                value="${DIR_GLOBALS}/ThirdParty/netty/4.1.1/netty-all-4.1.1.Final.jar"/>
+
+
+<!-- apache common -->
+<property name="JarCommonsIo"               value="${DIR_GLOBALS}/ThirdParty/commons-io/1.4/commons-io-1.4.jar"/>
+<property name="JarCommonsConfiguration"    value="${DIR_GLOBALS}/ThirdParty/commons-configuration/1.6/commons-configuration-1.6.jar"/>
+
+<!-- xstream -->
+<property name="JarXStream"               value="${DIR_GLOBALS}/ThirdParty/xstream/1.4.6/xstream-1.4.6.jar"/>
+
+
 <!-- Definition des IDL-Compilers: OpenORB ======================= -->
 
 <property name="idlj.classpath"             value="${JarOpenORB}:${JarOpenORBTools}:${JarOpenORBLogKit}:${JarOpenORBExcalibur}:${JarAvalonFramework}"/>
diff --git a/SelfServiceBuild/Build/java/build.xml b/SelfServiceBuild/Build/java/build.xml
index 44d15f0..10b2907 100644
--- a/SelfServiceBuild/Build/java/build.xml
+++ b/SelfServiceBuild/Build/java/build.xml
@@ -44,7 +44,8 @@
       ${PlatformJavaDir}/SOAP-Settings,
       ${PlatformJavaDir}/CLAApplet,
       ${PlatformJavaDir}/DSGEventReporting,
-      ${PlatformJavaDir}/Statechart"
+      ${PlatformJavaDir}/Statechart,
+      ${PlatformJavaDir}/BhsServer"
     />
     <property name="AblsDirList"     value="${AblsJavaDir}/KioskProxy,${AblsJavaDir}/TunnelServer"/>
     <property name="AppDirList"      value=""/>
-- 
2.41.0.windows.1

